tensorflow/core/grappler/optimizers/remapper_test.cc

```diff
@@ -742,14 +742,14 @@ TEST_F(RemapperFuseMatMulWithBiasAndActivationTest, F32) {
 }
 
 TEST_F(RemapperFuseMatMulWithBiasAndActivationTest, Bf16) {
-#if !defined(INTEL_MKL)
+#if !defined(ENABLE_MKL)
   GTEST_SKIP() << "Intel MKL with bfloat16 support is not enabled, skipping "
                   "FuseMatMulWithBiasAndActivation with bfloat16.";
-#endif                     // !defined(INTEL_MKL)
+#endif                     // !defined(ENABLE_MKL)
```

Comment:
Nit: Same here.
```suggestion
#endif  // !defined(ENABLE_MKL)
```

If this is the formatter's doing, please feel free to leave it like this.